ci-operator/config/maistra/istio-operator/maistra-istio-operator-maistra-3.0.yaml

```diff
@@ -139,25 +139,24 @@ tests:
       cli: latest
       commands: |
         export QUAY_PASS=$(cat /maistra-secrets/password)
+        export GITHUB_FILE=/creds-github/token
         # login to quay.io in the privileged pod
         oc rsh -n "${MAISTRA_NAMESPACE}" "${MAISTRA_SC_POD}" \
           docker login -u='maistra-dev+prow' \
           -p=${QUAY_PASS} quay.io
-        # build and push images
+        # build and push images and publish the bundle into OperatorHub
         oc rsh -n "${MAISTRA_NAMESPACE}" "${MAISTRA_SC_POD}" \
           entrypoint \
-          sh -c ' \
-          export REPO=quay.io/maistra-dev/istio-ubi9-operator; \
-          export IMAGE=$REPO:3.0-latest; \
-          make -e COMMUNITY=false clean image; \
-          docker tag $IMAGE $REPO:3.0-nightly-`date -I`; \
-          docker push $IMAGE; \
-          docker push $REPO:3.0-nightly-`date -I`'
+          sh -c \
+          "make -e GITHUB_FILE=${GITHUB_FILE} docker-push-nightly bundle-publish-nightly"
```

Comment:
I was thinking that `cat /creds-github/token` would display the token in the prow logs and especially because I use it in the [makefile](https://github.com/maistra/istio-operator/blob/maistra-3.0/Makefile#L286). So that's the reason I want to hide the content of this creds file using directly the file which will be processed by the script.

Reply:
In this CI, the Vault configuration will mask any secret content. The cat command will not display the token in this case.